src/Gui/Action.cpp

```diff
@@ -1166,6 +1167,20 @@ void RecentMacrosAction::setFiles(const QStringList& files)
         recentFiles[index]->setText(QString());
         recentFiles[index]->setToolTip(QString());
     }
+    // Raise a single warning no matter how many conflicts
+    if (!existingCommands.isEmpty()) {
+        auto msgMain = QStringLiteral("Recent macros : keyboard shortcut(s)");
+        for (int index = 0; index < accel_col.count(); index++) {
+            msgMain = msgMain + QStringLiteral(" %1").arg(accel_col[index]);
+        }
+        msgMain = msgMain + QStringLiteral(" disabled because conflicting with");
+        for (int index = 0; index < existingCommands.count(); index++) {
+            msgMain = msgMain + QStringLiteral(" %1").arg(existingCommands[index]);
+        }
+        msgMain = msgMain + QStringLiteral(" respectively.\nHint: In Preferences -> Macros -> Recent Macros -> Keyboard Modifiers"
+                                           " contains Ctrl+Shift+ as default, this may have been changed unintentionally.");

```

Comment:
I don't quite follow the wording here so I'm not sure which change to suggest. Can you explain the hint to me in more detail and I'll try to wordsmith something?